packages/fast-usdc-contract/src/exos/status-manager.ts

```diff
@@ -312,6 +318,9 @@ export const prepareStatusManager = (
         success: boolean,
         tx?: { destination: AccountId },
       ): void {
+        if (tx) {
+          routeHealth.noteSuccess(chainOfAccount(tx.destination));
+        }
```

Comment:
Does this need to branch since `success` can be `false`?

Reply:
Yes! Good catch. I'll fix this and try to make include a regression test

Reply:
And as you point out, it needs to be removed from `forwardFailed` if it's there. I'll make sure there's a test for that too